docs/compilation.md

```diff
@@ -41,6 +41,29 @@ If you're creating a package that relies on a custom build of `better-sqlite3`,
 
 If you're using a SQLite3 encryption extension that is a drop-in replacement for SQLite3 (such as [SEE](https://www.sqlite.org/see/doc/release/www/readme.wiki) or [sqleet](https://github.com/resilar/sqleet)), then simply replace `sqlite3.c` and `sqlite3.h` with the source files of your encryption extension.
 
+### Automated example
+Here's an automated script that will do all of this for you.
+
+```sh
+mkdir -p sqlite-amalgamation
+cd sqlite-amalgamation
+
+curl -O https://sqlite.com/2024/sqlite-amalgamation-3450100.zip
```

Comment:
Thanks for taking the time to write this PR!

Unfortunately, I suspect people will blindly copy and paste this in the future and not realize they're pulling in an out of date amalgamation.

If you want to be fancy, you can do something like this, which would be evergreen.

1. do the `npm install better-sqlite3` first.
2. then `mkdir -p sqlite-amalgamation && cd sqlite-amalgamation`
3. then something like this:

```
source <(egrep "^(YEAR|VERSION)=" ../node_modules/better-sqlite3/deps/download.sh)
curl https://sqlite.org/$YEAR/sqlite-amalgamation-$VERSION.tar.gz | tar -xz --strip 1
```
To fetch and extract into your new dir.

Reply:
That works for me!